node_modules/oae-core/userdetails/css/userdetails.css

```diff
@@ -0,0 +1,77 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+@media (min-width: 769px) and (max-width: 920px) {
+    #userdetails-modal .modal-dialog {
+        margin: 0 10px;
+        width: auto;
+    }
+}
+
+@media (min-width: 920px) {
+    #userdetails-modal .modal-dialog.termsandconditions {
+        width: 900px;
+    }
+}
+
+/*
+ * By default, show the user details form
+ */
+#userdetails-modal #userdetails-form {
+    display: block;
+}
+#userdetails-modal #userdetails-terms-and-conditions-container {
+    display: none;
+}
+#userdetails-modal .modal-footer {
```

Comment:
There should be one container for the main widget content (including its modal-footer) and one container for the terms and conditions (including its modal-footer). These should be toggled appropriately